docarray/documents/legacy/legacy_document.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+from typing import Any, Dict, Optional
+
+from docarray import BaseDocument, DocumentArray
+from docarray.typing import AnyEmbedding, AnyTensor
+
+
+class Document(BaseDocument):
+
+    tensor: Optional[AnyTensor]
+    chunks: Optional[DocumentArray[Document]]
+    matches: Optional[DocumentArray[Document]]
+    blob: Optional[Any]
+    text: Optional[str]
+    url: Optional[str]
+    embedding: Optional[AnyEmbedding]
+    # tags: Dict[str, Any] = dict()
```

Comment:
tags should be there

Reply:
Is there a way we can make the `oneof` behavior?

Reply:
should we implement `content`?

Reply:
as a property setter that checks where to put the value?

Reply:
>  Is there a way we can make the oneof behavior?

Oneof between what and what ?

Reply:
> should we implement content

I don't think it is nescerray tbh